dotnet/samples/KernelSyntaxExamples/Example09_FunctionTypes.cs

```diff
@@ -132,7 +132,7 @@ public string Type05(SKContext context)
     public async Task<string> Type06Async(SKContext context)
     {
         var summarizer = context.Functions.GetFunction("SummarizePlugin", "Summarize");
-        var summary = await context.Kernel.RunAsync("blah blah blah", summarizer);
+        var summary = await context.Kernel.RunAsync(summarizer, new ContextVariables("blah blah blah"));
```

Comment:
Do we have an obsolete method for the old method signature?

Reply:
There wasn't an old method, the previous `RunAsync` was from IKernel interface. As this is a recent change (that don't change the method name) 

I don't see a strong reason to create a new ExtensionClass implementing all the IKernel overloads Obsoleted into KernelExecutionContext.